lesson_1.1/src/InputWithLabel.js

```diff
@@ -0,0 +1,21 @@
+import React,{useEffect,useRef} from "react";
+
+
+function InputWithLabel({children,handleTitleChange,todoTitle}) { 
+    const inputRef = useRef ()
+    useEffect(() => { if(inputRef.current){inputRef.current.focus()}})
```

Comment:
if statement in useEffect isn't necessary. can be simplified to:
useEffect(() => { 
inputRef.current.focus() 
});